dm/loader/lightning.go

```diff
@@ -42,9 +38,9 @@ import (
 )
 
 const (
-	lightningCheckpointFile  = "lightning.checkpoint.0.sql"
-	lightningCheckpointDB    = "lightning"
-	lightningCheckpointTable = "checkpoint"
+	// checkpoint file name for lightning loader
+	// this file is used to store the real checkpoint data for lightning.
+	lightningCheckpointFileName = "tidb_lightning_checkpoint.pb"
```

Comment:
can you tell me why use `.pb` as checkponit file name?

Reply:
lightning file checkpoint is pb format